app-vcs/git-credential-oauth/autobuild/build

```diff
@@ -0,0 +1,7 @@
+abinfo "Building git-credential-oauth ..."
+cd "$SRCDIR"
```

Comment:
```suggestion
```

As per

https://github.com/AOSC-Dev/autobuild4/blob/ec0f29fa441e2a6a97bdfbd95a99b855bae304eb/templates/00-self.sh#L13-L18

I think we can also delete this line here?